jina/types/request/__init__.py

```diff
@@ -253,7 +253,7 @@ def parameters(self, value: Dict):
         self._pb_body.parameters.update(value)
 
 
-class Response:
+class Response(Request):
```

Comment:
I would add the Doc and GroundTruth mixin

Reply:
yep, you can do them in another pr